stations.csv

```diff
@@ -35269,3 +35269,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 38105;Kiev Boryspil Airport;kiev-boryspil-airport;;;50.341475;30.889526;32270;UA;Europe/Kiev;f;f;t;t;f;f;;;f;;f;;f;;f;UAIEVKA;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;Kyiw;;;;;;;Kyjev;;Kijev;キエフ;키예프;Kijów;;Киев;;;;urn:trainline:public:nloc:csv38105
 38106;Ciboure;ciboure;;;43.383354;-1.690519;;FR;Europe/Paris;t;f;f;t;f;f;;;f;;f;;f;;f;FRCIB;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv38106
 38107;St. Andrä-Wördern;st-andra-wordern;;;48.333936;16.209135;;AT;Europe/Vienna;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;8100432;t;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv38107
+38108;Nîmes Pont du Gard;nimes-pont-du-gard;8770397;87703975;43.8169444;4.507777777777778;22600;FR;Europe/Paris;f;f;f;f;f;f;FRXZP;;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;FRXZP;f;;f;t;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv38108
```

Comment:
You can make the longitude shorter, we don't need nanometer (or smaller!) precision. You can shorten it to 5 or 6 decimal places.